tests/unit/test_model_card.py

```diff
@@ -701,6 +797,32 @@ def test_create_model_card(
     assert card.arn == MODEL_CARD_ARN
 
 
+@patch("sagemaker.Session")
+def test_create_model_card_with_model_package(
+    session,
+    model_package_example,
+    intended_uses_example,
+    business_details_example,
+    additional_information_example,
+):
+    session.sagemaker_client.create_model_card = Mock(return_value=CREATE_MODEL_CARD_RETURN_EXAMPLE)
+    session.sagemaker_client.describe_model_card = Mock(return_value=LOAD_MODEL_CARD_EXMPLE)
```

Comment:
you will need to create a model card object with content that including all the data in the `model_package_example` ([it is a call right after creater](https://github.com/shrestha-bikash/sagemaker-python-sdk/blob/79cfb94b80b8817386c9216c3a329803afc1a05e/src/sagemaker/model_card/model_card.py#L915))

For simplicity, you could remove intended_uses, business_details, additional_information so that the `Content` field only contain information related to model package in the describe_model_package call.

(The existing tests `create_model_card` has the same problem, I just saw...we could fix that later)

Reply:
Yeah I noticed that. The current test `test_create_model_card` does not test for other contents. It just checks for model card ARN. I thought we were just checking for that and did not bother to create another mock response for model package stuff. 

I will add the mock response for model package content and will add test them.